org-code-javabuilder/protocol/src/main/java/org/code/protocol/JavabuilderThrowableMessageUtils.java

```diff
@@ -31,15 +41,17 @@ public static HashMap<String, String> getExceptionDetails(
 
     String preferredFallbackMessage = null;
     if (cause != null) {
-      // To do: remove CAUSE key once we're using STACK_TRACE and EXCEPTION_TYPE
-      // in (partially) translated exception messages in Javalab.
+      // To do: remove CAUSE key and replace with stack trace and exception message individually
+      // to enable (partially) translated exception messages in Javalab.
       // https://codedotorg.atlassian.net/browse/JAVA-439
       preferredFallbackMessage = getDefaultFallbackMessageString(cause);
       detail.put(ClientMessageDetailKeys.CAUSE, preferredFallbackMessage);
-      detail.put(ClientMessageDetailKeys.STACK_TRACE, getUserFacingStackTraceString(cause));
```

Comment:
`STACK_TRACE` and `EXCEPTION_MESSAGE` were added to allow us to translate part of the exception message to students, and are not used by Java Lab. Since these make the payload much larger, removing them for now (until we actually want to do that translation work).

https://github.com/code-dot-org/javabuilder/pull/205